### DISCUSSION TITLE
Cursos focados em ReactJS !

### DISCUSSION THREAD

**Discussion by PetrusBorges** [frontendbr/forum#2329] [Discussão]
Fala manos, estou começando no React, e gostaria da indicação de algum curso focado nele, com tantas opções fico perdido em qual realmente investir.

**Comment by romulociro** [comment]
Recomendo os cursos da Rocketseat , aprendi muito com eles.

**Comment by felquis** [comment]
Recomendo ver outros tópicos com a mesma pergunta https://github.com/frontendbr/forum/discussions?discussions_q=curso+react
